packages/server/src/models/user.entity.ts

```diff
@@ -13,4 +13,10 @@ export default class User {
 
```

Comment:
I'd suggest use firstName and lastName instead of name

Reply:
Facebook returns us only `displayName`. Would that go?

Reply:
can we split it by space?

Reply:
Sorry, just have investigated a bit more and found out that all the fields can be extracted from a Facebook account. Don't you mind if the changes would be added during the actual implementation of `passport-facebook` `passport-google`?

Reply:
If it not break anything after (migration, db changes, created user), I'm ok